src/OrchardCore/OrchardCore.ResourceManagement.Abstractions/ResourceRequiredContext.cs

```diff
@@ -16,9 +19,27 @@ public IHtmlContent GetHtmlContent(string appPath)
         {
             var tagBuilder = Resource.GetTagBuilder(Settings, appPath, FileVersionProvider);
 
+            var attributes = new TagHelperAttributeList();
+            foreach (var attribute in tagBuilder.Attributes)
+            {
+                if (attribute.Key != "integrity")
+                {
+                    attributes.Add(attribute.Key, attribute.Value);
```

Comment:
That's so bad ;) Can't we pass an HtmlString as the value instead? Just to tell this tag that this shouldn't be encoded?

Reply:
I mean hard-coding the name here is bad.

Reply:
Yes i know, i already did it at a lower level but with more changes (i know you prefer fewer changes) and then **here without using a `TagBuilder line 20`**. This because the tag builder holds attribute values a strings, not as objects that could be e.g an `HtmlString`. So yes, we could use the `TagHelperOutput` (whose attribute values are objects) at a lower level but with a few more changes.